components/menteeDetails/BubbleList.js

```diff
@@ -31,4 +21,14 @@ class BubbleList extends Component {
   }
 }
 
+const styles = StyleSheet.create({
```

Comment:
Having these consistent and at the bottom makes this code so much better :) 

Is there an eslint rule to enforce this practice?

Reply:
I tried to look for one but I didn't find any